app/models/edition.rb

```diff
@@ -30,6 +30,9 @@ class Edition < ActiveRecord::Base
   has_many :classification_featurings, inverse_of: :edition
   has_many :links_reports, as: :link_reportable
 
+  has_many :dependencies, class_name: 'EditionDependency', foreign_key: :edition_id, dependent: :destroy
```

Comment:
Sorry to chime in late on this one, but I'm really uncomfortable with having an association called `edition_dependencies` that represents the other end of a join model on a table called `edition_dependencies` - it seems like it would create much scope for confusion. I have a potential alternative solution - could we use a scope on the `EditionDependency` model to allow the type of dependency to be selected rather than explicitly defining associations for them? e.g.

```ruby

class Edition
  has_many :edition_dependencies, dependent: :destroy
  has_many :depedencies, through: :edition_dependencies, source: :dependable
end

class EditionDependency
  scope :editions, -> { where(type: 'Edition') }
  scope :contacts, -> { where(type: 'Contact') }
end

# which would allow:
edition.dependencies.contacts
edition.dependencies.editions
```